modules/integration/tests-integration/tests/src/test/java/org/wso2/appserver/integration/tests/carbontools/ChangeUserPasswordMySQLDBTestCase.java

```diff
@@ -0,0 +1,113 @@
+/*
+ *
+ *  * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *  *
+ *  * WSO2 Inc. licenses this file to you under the Apache License,
+ *  * Version 2.0 (the "License"); you may not use this file except
+ *  * in compliance with the License.
+ *  * You may obtain a copy of the License at
+ *  *
+ *  * http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing,
+ *  * software distributed under the License is distributed on an
+ *  * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  * KIND, either express or implied.  See the License for the
+ *  * specific language governing permissions and limitations
+ *  * under the License.
+ *
+ */
+
+package org.wso2.appserver.integration.tests.carbontools;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.appserver.integration.common.utils.ASIntegrationTest;
+import org.wso2.appserver.integration.common.utils.CarbonCommandToolsUtil;
+import org.wso2.carbon.automation.engine.context.AutomationContext;
+import org.wso2.carbon.automation.engine.context.ContextXpathConstants;
+import org.wso2.carbon.automation.extensions.servers.carbonserver.TestServerManager;
+import org.wso2.carbon.integration.common.admin.client.AuthenticatorClient;
+import org.wso2.carbon.integration.common.extensions.usermgt.UserPopulator;
+
+import java.io.File;
+
+/**
+ * This class is to test change MySQL user password using chpasswd.sh/chpasswd.bat
+ */
+public class ChangeUserPasswordMySQLDBTestCase extends ASIntegrationTest {
+
+    private static final Log log = LogFactory.getLog(ChangeUserPasswordH2DBTestCase.class);
+    public static String mySQL_DB_URL ;
+
+    private TestServerManager testServerManager;
+    private AutomationContext context;
+    private AuthenticatorClient authenticatorClient;
+    private boolean scriptRunStatus;
+
+    @BeforeClass(alwaysRun = true)
+    public void init() throws Exception {
+        context = new AutomationContext("AS", "appServerInstance0002", ContextXpathConstants.SUPER_TENANT,
+                                        ContextXpathConstants.ADMIN);
+        authenticatorClient = new AuthenticatorClient(context.getContextUrls().getBackEndUrl());
+        authenticatorClient = new AuthenticatorClient(context.getContextUrls().getBackEndUrl());
+        String mySQLPath  = "//databasePath/MySQL";
```

Comment:
Create a framework constant class to keep all automation .xml related xpath values